src/test/java/me/aborozdykh/amazonreview/DataToDbSaverTest.java

```diff
@@ -0,0 +1,126 @@
+        dataToDbSaver.saveReviewsToDb(getReviewRequestDtoList());
+        var reviewRequestDto = getReviewRequestDtoList().get(0);
+        var review = reviewRepository.getOne(reviewRequestDto.getId());
+
+        Assert.assertEquals(reviewRequestDto.getId(), review.getId());
+        Assert.assertEquals(reviewRequestDto.getDateTime(), review.getDateTime());
+        Assert.assertEquals(reviewRequestDto.getHelpfulnessDenominator(), review.getHelpfulnessDenominator());
+        Assert.assertEquals(reviewRequestDto.getHelpfulnessNumerator(), review.getHelpfulnessNumerator());
+        Assert.assertEquals(reviewRequestDto.getScore(), review.getScore());
+        Assert.assertEquals(reviewRequestDto.getSummary(), review.getSummary());
+        Assert.assertEquals(reviewRequestDto.getText(), review.getText());
+    }
+
+    @Test
+    public void saveToDbIsOk(){
+        dataToDbSaver.saveToDb(getReviewRequestDtoList());
+        var reviewRequestDto = getReviewRequestDtoList().get(0);
+        var user = userRepository.getOne(reviewRequestDto.getUserId());
+        var product = productRepository.getOne(reviewRequestDto.getProductId());
+        var review = reviewRepository.getOne(reviewRequestDto.getId());
+
+        Assert.assertEquals(reviewRequestDto.getUserId(), user.getId());
+        Assert.assertEquals(reviewRequestDto.getProfileName(), user.getProfileName());
+        Assert.assertEquals(reviewRequestDto.getProductId(), product.getId());
+        Assert.assertEquals(reviewRequestDto.getId(), review.getId());
+        Assert.assertEquals(reviewRequestDto.getDateTime(), review.getDateTime());
+        Assert.assertEquals(reviewRequestDto.getHelpfulnessDenominator(), review.getHelpfulnessDenominator());
+        Assert.assertEquals(reviewRequestDto.getHelpfulnessNumerator(), review.getHelpfulnessNumerator());
+        Assert.assertEquals(reviewRequestDto.getScore(), review.getScore());
+        Assert.assertEquals(reviewRequestDto.getSummary(), review.getSummary());
+        Assert.assertEquals(reviewRequestDto.getText(), review.getText());
+    }
+
+    private List<ReviewRequestDto> getReviewRequestDtoList() {
+        String id = "1";
+        String productId = "B001E4KFG0";
+        String userId = "A3SGXH7AUHU8GW";
+        String profileName = "delmartian";
+        String helpfulnessNumerator = "1";
+        String helpfulnessDenominator = "1";
+        String score = "5";
+        String dateTime = "1303862400";
+        String summary = "Good Quality Dog Food";
+        String text = "1,B001E4KFG0,A3SGXH7AUHU8GW,delmartian,1,1,5,1303862400,Good Quality Dog "
+                + "Food,I have bought several of the Vitality canned dog food products and have "
+                + "found them all to be of good quality. The product looks more like a stew than"
+                + " a processed meat and it smells better. My Labrador is finicky and she "
+                + "appreciates this product better than  most.";
```

Comment:
let's cut out the majority of text from it's review(and from test file too). We don't need the full text for test.